packages/frontend-api/src/Resources/config/graphql-types/AdvertCodeDecorator.types.yaml

```diff
@@ -0,0 +1,12 @@
+AdvertCodeDecorator:
+    type: object
+    decorator: true
+    inherits:
+        - 'AdvertDecorator'
+    config:
+        fields:
+            code:
+                type: "String"
```

Comment:
Code is mandatory in administration. Is it here nullable intentionally?

Reply:
fixed in 2c1eada980f8c258b2dc6fc85c43a79f020bfe28